### PR TITLE
[3.3.5][6.x] Core/Spells: Fix wrong totem placement

### PR DESCRIPTION
**Pull request by Eliminationzx**
Closes https://github.com/TrinityCore/TrinityCore/issues/14888

### CODE REVIEW

File: src/server/game/Spells/Spell.cpp [hunk 1]
```diff
@@ -1288,18 +1288,30 @@ void Spell::SelectImplicitCasterDestTargets(SpellEffIndex effIndex, SpellImplici
...
+                    dist = PET_FOLLOW_DIST;
+                    break;
+                case TARGET_DEST_CASTER_RANDOM:
+                    dist = objSize + (dist - objSize) * float(rand_norm());
+                    break;
+                case TARGET_DEST_CASTER_FRONT_LEFT:
+                case TARGET_DEST_CASTER_BACK_LEFT:
+                case TARGET_DEST_CASTER_FRONT_RIGHT:
+                case TARGET_DEST_CASTER_BACK_RIGHT:
+                    dist = objSize * 2.0f;
```
Comment (joschiwald) [inline src/server/game/Spells/Spell.cpp]: you should not ignore radius if it exists

i don't think `objSize * 2`is a good fallback

**Comment by Eliminationzx** [comment]
bump

**Comment by Eliminationzx** [comment]
any feedback?

**Comment by MitchesD** [comment]
I am still not sure about that, need some confirmation from other guys

**Comment by Kittnz** [comment]
No more feedback, merging.

**Comment by Kittnz** [comment]
Thanks for the PR!

**Pull request merged**